### DISCUSSION TITLE
investigation

### DISCUSSION THREAD

**Discussion by otestotestda22** [bagisto/bagisto#9288] [Q&A]
could you tell me the use of the "send check to" field on the money transfer payment method page or how it should work 
shown in attachment below
![money transfer](https://github.com/bagisto/bagisto/assets/155967365/150efa77-b5ae-4136-a051-a4ff81a0d4e6)

**Comment by amit-webkul** [comment]
Dear @otestotestda22,

I have investigated your inquiry regarding the "Send check to" field in the money transfer configuration section and have identified its purpose. This field serves to include a message concerning payment. Once a message is added, the outcome is displayed within the order payment section.

In summary, this feature primarily facilitates money order payment mode by providing a space for including details pertinent to the money order.

In case you have any doubts, feel free to reach out to us.
